owncloudApp/src/main/java/com/owncloud/android/ui/activity/ToolbarActivity.kt

```diff
@@ -137,4 +145,21 @@ abstract class ToolbarActivity : BaseActivity() {
     private fun getRootToolbar(): ConstraintLayout = findViewById(R.id.root_toolbar)
 
     private fun getStandardToolbar(): Toolbar = findViewById(R.id.standard_toolbar)
+
+    override fun onPrepareOptionsMenu(menu: Menu): Boolean {
+        (menu.findItem(R.id.action_search).actionView as SearchView).run {
+            val searchText = findViewById<EditText>(androidx.appcompat.R.id.search_src_text)
+            val closeButton = findViewById<ImageView>(androidx.appcompat.R.id.search_close_btn)
+            val searchButton = findViewById<ImageView>(androidx.appcompat.R.id.search_button)
+
+            maxWidth = Int.MAX_VALUE
+
+            searchButton.setBackgroundColor(getColor(R.color.actionbar_start_color))
```

Comment:
Why is this needed just for the standard toolbar and not the root toolbar? Or maybe it is not needed at all?

Reply:
In this case, the the background color of the searchButton get the color of the background SearchView, so we have to set the color of the toolbar.